spp_area/models/area_import.py

```diff
@@ -106,10 +106,15 @@ def import_data(self):  # noqa: C901
                     for col in range(sheet.ncols):
                         col_name = sheet.cell(0, col).value
                         if col_name.find("Name") >= 0:
-                            maxcolsstr = col_name.strip("admin").replace("Name", "")
+                            maxcolsstr = ""
+                            if col_name.find("admin") >= 0:
+                                maxcolsstr = col_name.strip("admin").replace("Name", "")
+                            elif col_name.find("Area") >= 0:
+                                maxcolsstr = col_name.strip("Area").replace("Name", "")
```

Comment:
What is the Area Column?

Reply:
Its the new prefix for the new template. But it was fixed now that it will be also compatible with the template we used before.